code/style.css

```diff
@@ -1,4 +1,335 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+/** over all styling **/
+
+body {
+  background-color: #f0f0f0;
+  color: #242323;
+  margin: 0 auto;
+  padding: 0;
+  padding-top: 70px;
+  width: 100%;
+  font-family: Georgia, 'Times New Roman', Times, serif
+ }
+ 
+ a {
+   text-decoration: none;
+   color: rgb(59, 58, 58);
+ }
+
+ 
+ /** styling of header & navigation **/
+ 
+ .flex-container {
+   display: flex;
+   align-items: center;
+   justify-content: space-evenly;
+   background-color: #f0f0f0;
+   width: 100%;
+   margin: 0;
+   padding: 0;
+   position: fixed;
+   top: 0;
+ }
+ 
+ .logo {
+  display: flex;
+  flex-direction: row;
+  width: 150px;
+  
 }
+
+.hidden-anchor {    /* the hidden anchor is needed to keep the margin between the fixed header and the first big-news-article*/
+  height: 90px;
+}
+ 
+
+ h1 {
+   display: block;
+   font-family: Georgia, 'Times New Roman', Times, serif;
+   font-size: 20px;
+   margin-left: 20px;
+ }
+ 
+ nav {
+   display: none;
+ }
+ 
+ nav a {
+   text-decoration: none;
+   font-size: 10px;
+ }
+ 
+ nav a:hover {
+   font-weight: bold;
+ }
+ 
+ /** styling of big news section **/
+ 
+ .big-news {
+   background-color: #fff;
+   display: flex;
+   flex-direction: column;
+   justify-content: center;
+   align-items: center;
+   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+   margin-top: 10px;
+ }
+ 
+ .big-news-image {
+   margin: 20px;
+   width: 90%;
+ }
+ 
+ .big-news:hover {
+   background-color: #fafafa;
+ }
+ 
+ h2 {
+   margin-top: 10px;
+   font-size: 24px;
+ }
+ 
+ .big-news p {
+   font-size: 20px;
+   margin-bottom: 30px;
+ }
+ 
+ .news-text {
+   text-align: center;
+   margin: 0 20px 30px 20px;
+ }
+ 
+ .news-text a:hover {
+   font-weight: bold;
+ }
+ 
+ .read-link {
+   background-color: #848686;
+   color: #fff;
+   font-size: 20px;
+   padding: 2px 8px;
+   border-radius: 20px;
+ }
+ 
+ /** styling of card container **/
+ 
+ h3 {
+   font-size: 20px;
+   margin: 20px 0 -10px;
+ }
+ .card-container p {
+   font-size: 18px;
+ }
+ 
+ .card-container {
+   display:flex;
+   flex-direction: column;
+   text-align: center; /* for the h3 & p*/
+   align-items: stretch;
+ }
+ 
+ .cards {
+   display: flex;
+   flex-wrap: wrap;
+   flex-direction: column;
+   align-items: center;
+   justify-content: center;
+   
+ }
+ 
+ .cards a {
+   background: #fff;
+   width: 90%;
+   box-shadow: 5px 5px 0 0 #E0E7E9;
+   margin: 10px 5px;
+ }
+ 
+ .card-image {
+   margin: 20px 5px;
+   width: 90%;
+ }
```

Comment:
The images are on left side of the container and the text isn't inline. Did you do this out of design reasons? Otherwise you could try this code to get everything inline 😃 

```
.card-image {
    margin: 20px auto;
    width: 90%;
    display: block;
}

.card-text {
    margin: 10px auto;
    text-align: center;
    width: 90%;
}
```

Reply:
Thank you, I will try this :)